_posts/en/newsletters/2023-01-18-newsletter.md

```diff
@@ -0,0 +1,216 @@
+      to an ultimate destination.  Such proposals also generally require
+      each received output be spent in a separate transaction from other
+      received outputs, preventing the use of [payment batching][topic
+      payment batching].  This increases the number of transactions
+      involved, which also bloats the size and cost of spending.
+
+      `OP_VAULT` requires fewer transactions when spending a single
+      output in typical cases and it also supports batching when
+      spending or freezing multiple outputs, potentially saving a large
+      amount of space and allowing vaults to receive many more
+      transactions before their outputs would need to be consolidated
+      for safety.
+
+    In discussion of the idea, Greg Sanders proposed (as [summarized by
+    O'Beirne][obeirne scripthash]) a slightly different construction which
+    "would allow all outputs in vault lifecycles to be [P2TR][topic
+    taproot], for example, which would conceal the operation of the
+    vault---a very nice feature".
+
+    Separately, Anthony Towns [notes][towns op_vault] that the proposal
+    allows the vault user to freeze their funds at any time just by
+    spending the funds to the address for the highly trusted spending
+    path, allowing the user to unfreeze their funds later.  This
+    benefits the vault owner because they don't need to access any
+    specially-secured key material, such as a cold wallet, in order to
+    block a theft attempt.  However, any third party who learns the
+    address can also freeze the user's funds (although they'll have to
+    pay a transaction fee to do so), creating an inconvenience for the
+    user.  Given that many lightweight wallets disclose their addresses
+    to third parties in order to locate their onchain transactions,
+    vaults built on those wallets might unintentionally give third
+    parties the ability to grief vault users.  Towns suggests an
+    alternative construction for the freezing condition that would
+    require providing an additional piece of non-private information in
+    order to initiate a freezing, preserving the benefits of the scheme
+    but also reducing the risk that a wallet would unnecessarily give
+    third parties the ability to freeze funds and grief the user.  Towns
+    also suggests a possible improvement in batching support and
+    contemplates taproot support.
+
+    Several replies also mentioned that `OP_UNVAULT` can provide many of
+    the features of the [OP_CHECKTEMPLATEVERIFY][topic
+    op_checktemplateverify] (CTV) proposal, including the benefits to
+    [DLCs][topic dlc] previously described in [Newsletter #185][news185
+    ctv-dlc], although at greater onchain cost than directly using CTV.
+
+    As of this writing, discussion on the mailing list was no longer
+    active but we expect the idea will continue to be evaluated
+    alongside other covenant and vault proposals.
```

Comment:
I guess this could be changed to "discussion still active" given yesterday's messages?